elide-contrib/elide-dynamic-config-helpers/src/main/java/com/yahoo/elide/contrib/dynamicconfighelpers/validator/DynamicConfigValidator.java

```diff
@@ -0,0 +1,267 @@
+/*
+ * Copyright 2020, Yahoo Inc.
+ * Licensed under the Apache License, Version 2.0
+ * See LICENSE file in project root for terms.
+ */
+package com.yahoo.elide.contrib.dynamicconfighelpers.validator;
+
+import com.yahoo.elide.contrib.dynamicconfighelpers.DynamicConfigHelpers;
+import com.yahoo.elide.contrib.dynamicconfighelpers.model.Dimension;
+import com.yahoo.elide.contrib.dynamicconfighelpers.model.ElideSecurityConfig;
+import com.yahoo.elide.contrib.dynamicconfighelpers.model.ElideTableConfig;
+import com.yahoo.elide.contrib.dynamicconfighelpers.model.Join;
+import com.yahoo.elide.contrib.dynamicconfighelpers.model.Measure;
+import com.yahoo.elide.contrib.dynamicconfighelpers.model.Table;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+import org.apache.commons.io.FileUtils;
+
+import lombok.Data;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.Locale;
+import java.util.Map;
+import java.util.Set;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@Slf4j
+@Data
+/**
+ * Util class to validate the model config files in local before deployment.
+ */
+public class DynamicConfigValidator {
+
+    private static final Set<String> SQL_DISALLOWED_WORDS = new HashSet<>(
+            Arrays.asList("DROP", "TRUNCATE", "DELETE", "INSERT", "UPDATE", "ALTER", "COMMENT", "CREATE", "DESCRIBE",
+                    "SHOW", "USE", "GRANT", "REVOKE", "CONNECT", "LOCK", "EXPLAIN", "CALL", "MERGE", "RENAME"));
+    private static final String[] ROLE_NAME_DISALLOWED_WORDS = new String[] { "," };
+    private static final String SQL_SPLIT_REGEX = "\\s+";
+    private static final String SEMI_COLON = ";";
+    private static final Pattern HANDLEBAR_REGEX = Pattern.compile("<%(.*?)%>");
+
+    private ElideTableConfig elideTableConfig;
+    private ElideSecurityConfig elideSecurityConfig;
+    private Map<String, Object> variables;
+    private String configDir;
+
+    public static void main(String[] args) throws IOException, ParseException {
+
+        Options options = new Options();
+        defineArgs(options);
+        usage(options);
```

Comment:
instead of printing usage always, lets do -h/--help option for printing the usage on demand.

Reply:
sure